tensorflow/lite/python/lite_v2_test.py

```diff
@@ -2213,6 +2213,48 @@ def testKerasConvLSTM2DWithMoreThanOneDilationRate(self):
     tflite_model = converter.convert()
     self.assertTrue(tflite_model)
 
+  @test_util.run_v2_only
+  def testKerasFullyConnectedQATHasNoReshapes(self):
+    """Create a simple QAT FullyConnected Model to ensure reshapes are removed."""
+    input_tensor = tf.keras.layers.Input(
+        batch_size=1,
+        shape=[8, 4],
+        name='input_tensor',
+        dtype=tf.float32)
+
+    x = tf.quantization.fake_quant_with_min_max_args(input_tensor, -3.0, 3.0)
+    x = tf.keras.layers.Dense(16)(x)
+    x = tf.quantization.fake_quant_with_min_max_args(x, -3.0, 3.0)
+    model = tf.keras.Model(input_tensor, x)
```

Comment:
Does this leave a reshape without your changes?

Reply:
Hi, 
Thanks for review. Yes, without the change a reshape will remain before and after the FullyConnected layer.
Best Regards,
Saoirse

Reply:
Looks good to me, but a couple of linting issues:

tensorflow/lite/python/lite_v2_test.py:2370:0: C0301: Line too long (87/80) (line-too-long)
tensorflow/lite/python/lite_v2_test.py:3111:0: C0301: Line too long ([9](https://github.com/tensorflow/tensorflow/pull/55205/checks#step:7:9)6/80) (line-too-long)
tensorflow/lite/python/lite_v2_test.py:3162:0: C0301: Line too long (96/80) (line-too-long)
tensorflow/lite/python/lite_v2_test.py:3188:0: C0301: Line too long (96/80) (line-too-long)

Reply:
Thank you. 
I have fixed the issues that were in the new test, but left the ones above as these are not part of the new test. 
I am happy to fix these too, if you would like though.
Best regards,
Saoirse